doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -9,6 +9,43 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+liblwm2m_carrier 0.22.0
+***********************
+
+Release for modem firmware version 1.3.1.
+
+Certification status
+====================
+
+For certification status, see `Mobile network operator certifications`_.
+
+Size
+====
+
+See :ref:`lwm2m_lib_size` for an explanation of the library size in different scenarios.
+
++-------------------------+---------------+------------+
+|                         | Flash (Bytes) | RAM (Bytes)|
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (binary)                |               |            |
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (reference application) |               |            |
++-------------------------+---------------+------------+
```

Comment:
I guess the sizes will be added to the table at some point.

Reply:
Yes, closer to merge :)